articles/quickstart.md

```diff
@@ -1,163 +1,129 @@
-- Make sure you have `.NET Framework 4.6.1` selected in the list at the bottom of the `New Project` dialog box.
-- Give your project the name `Bell`.
+1. Create a new project
 
-***
+   * Open Visual Studio
+   * Go to the **File** menu and select **New** -> **Project...**
+   * In the project template explorer, under `Installed` > `Visual C#`, select the `Q# Application` template
+   * Make sure you have `.NET Framework 4.6.1` selected in the list at the bottom of the `New Project` dialog box.
+   * Give your project the name `Bell`
 
-### Step 2: Enter the Q# Code
+1. Rename the Q# file
 
-Our goal is to create a [Bell State](https://en.wikipedia.org/wiki/Bell_state) showing entanglement. We will build this up piece by piece to show the concepts of qubit state, gates and measurement.
+   * Navigate to the **Solution Explorer**
+   * Right click on the `Operation.qs` file
+   * Rename it to `Bell.qs`
 
-#### Source files for a quantum application
-Your development environment should have two files open:
-`Driver.cs`, which will hold the C# driver for your quantum code,
-and `Operations.qs`, which will hold the quantum code itself.
+* * *
 
-#### Q# source
-The first step is to rename the Q# file to `Bell.qs`.
-Right-click on `Operations.qs` in the Visual Studio Solution Explorer (Ctrl+Alt+L to focus) or the Visual Studio Code Explorer (Ctrl/⌘+Shift+E to focus), and select the Rename option.
-Replace `Operations` with `Bell` and hit return.
+## Write a Q# operation
 
-To enter the Q# code, make sure that you are editing the
-`Bell.qs` window.
-The `Bell.qs` file should have the following contents:
+Our goal is to create a [Bell State](https://en.wikipedia.org/wiki/Bell_state) showing entanglement. We will build this up piece by piece to show the concepts of qubit state, gates, and measurement.
 
-```qsharp
-namespace Quantum.Bell {
-    open Microsoft.Quantum.Intrinsic;
-    open Microsoft.Quantum.Canon;
+### Q# operation code
+
+1. Replace the contents of the Bell.qs file with the following code:
+
+    ```qsharp
+    namespace Quantum.Bell {
+        open Microsoft.Quantum.Intrinsic;
+        open Microsoft.Quantum.Canon;
 
-    operation HelloQ () : Unit {
-        Message("Hello quantum world!");
+        operation Set (desired: Result, q1: Qubit) : Unit {
+            if (desired != M(q1)) {
+                X(q1);
+            }
+        }
     }
-}
-```
+    ```
 
-#### Q# operation
-First, replace the string `HelloQ` with `Set`, and change the operation
-parameters (the content of the parentheses) to contain the string
-`desired: Result, q1: Qubit`.
-The file should now look like:
+    This operation may now be called to set a qubit to a known state (`Zero` or `One`).
 
-```qsharp
-namespace Quantum.Bell {
-    open Microsoft.Quantum.Intrinsic;
-    open Microsoft.Quantum.Canon;
+    We measure the qubit, if it's in the state we want, we leave it alone, otherwise, we flip it with the `X` gate.
```

Comment:
Do we want `X` to be a link to the API reference for Microsoft.Quantum.Intrinsic.X?

Reply:
I like this, but I'm not sure that API reference tells a new user anything too interesting beyond operation arguments? (which might be a sufficient amount of information)